requirements.txt

```diff
@@ -1,7 +1,20 @@
+alembic
+beautifulsoup4
+bunch
 Flask
+Flask-WTF
+itsdangerous
+Jinja2
```

Comment:
Hmm, no, we need to track our dependencies, not just list what pip installed at some point in time. Using `pip freeze` means even more work for us, since we'd need to track all our dependencies' dependencies: if for example if Flask remove one of its dependencies, we would never know if we use pip freeze, and we'd keep requiring it even if it's not actually needed.
(btw, that would be quite ironic coming from a package reviewing app to not track its dependencies properly ;-) )